src/ducks/context/JobsContext.spec.jsx

```diff
@@ -42,7 +53,7 @@ describe('Jobs Context', () => {
       <JobsContext.Consumer>
         {({ jobsInProgress }) => {
           return jobsInProgress.map(job => (
-            <div key={job.account}>
+            <div key={job.konnector}>
```

Comment:
Pourquoi ce changement ?

Reply:
On n'affiche qu'un job in progress par connecteur et de plus, les nouveau jobs d'attente de job n'ont pas d'account associé mais ont bien un connecteur